src/jumio-kyc/kyc.service.spec.ts

```diff
@@ -46,6 +48,11 @@ describe('KycService', () => {
     await app.init();
   });
 
+  beforeEach(async () => {
+    await prisma.redemption.deleteMany();
+    await prisma.userPoints.deleteMany();
```

Comment:
you don't need this if your allocation code can just filter on a subset of users, and it will make sure tests don't get flakey.